tasks/yum.yml

```diff
@@ -9,18 +9,29 @@
   with_items:
     - sudo
     - gcc
-    - sqlite
     - supervisor
     - nginx
     - openssl
     - openssl-devel
     - libffi-devel
     - python-devel
+    - redis
     - python-firewall
     - libsemanage-python
     - policycoreutils-python
     - redhat-rpm-config
 
+- name: Install spatialite
+  when: openwisp2_database.engine == "django.contrib.gis.db.backends.spatialite"
+  apt: name={{ item }} state=latest
+  notify: reload systemd
+  with_items:
+    - sqlite
+    - gdal
+    - libproj-devel
```

Comment:
Didn't find this. But found `libprojectM-devel`, not sure is this the same?

Reply:
Found this while googling: http://rpm.pbone.net/index.php3/stat/4/idpl/29688291/dir/centos_6/com/libproj-devel-4.8.0-24.1.x86_64.rpm.html
Isn't it available on the system you use?

Here's another one named `proj-devel`:
https://centos.pkgs.org/7/epel-x86_64/proj-devel-4.8.0-4.el7.x86_64.rpm.html

Reply:
I was looking only for CentOS7. And yes, `proj-devel` is available indeed for CentOS7 and CentOS6.